Cargo.toml

```diff
@@ -9,12 +9,13 @@ edition = "2021"
 resolver = "2"
 members = [
     "src/binaries",
-    "src/common/arrow",
     "src/common/auth",
     "src/common/base",
     "src/common/building",
     "src/common/cache",
     "src/common/cloud_control",
+    "src/common/column",
+    "src/common/native",
```

Comment:
How about using `strawboat` instead to avoid confusing `native`?

Reply:
native is based on databend's column, it's different with opensource strawboat now.